source/JointMilitarySymbologyLibraryCS/jmsml/test/Fields_ControlMeasuresAreas.csv

```diff
@@ -0,0 +1,26 @@
+field_name,field_type,field_length,field_alias,nullability,field_domain
+identity,Integer,4,Standard Identity,NULLABLE,Identity
+symbolset,SmallInteger,2,Symbol Set,NULLABLE,
+entity,Integer,4,Control Measures Area,NULLABLE,Control_Measure_Area_Entities
+echelon,Integer,4,echelon,NULLABLE,Echelons
+uniquedesignation,String,50,Unique Designation,NULLABLE,
+additionalinformation,String,50,Additional Information,NULLABLE,
+additionalinformation2,String,50,Additional Information 2,NULLABLE,
+hostile,String,3,Hostile (ENY),NULLABLE,
+type,String,50,Type of Equipment,NULLABLE,
```

Comment:
Sure.  The Type field was variously 24, 50, or 255 in different baseline files.  In order to better normalize the fields across all the files, including their relative order, so that the files are more consistent, I settled on a value for items like length and stuck with that value throughout all the schemas.  Even if Type is listed with different display lengths in 2525 or APP-6 in various appendices/chapters, those are only display lengths, not necessarily the length of the corresponding data in a database or in a message.  A further reminder that the symbology standard is a display standard and not a database standard.

If there remains a good reason to have two different Type fields in the geodatabase, one with length 24 and another with length 50, I can certainly amend the JMSML data to accommodate that.